bin/millau/node/src/chain_spec.rs

```diff
@@ -70,10 +70,7 @@ impl Alternative {
 		let properties = Some(
 			serde_json::json!({
 				"tokenDecimals": 9,
-				"tokenSymbol": "MLAU",
-				"bridgeIds": {
-					"Rialto": bp_runtime::RIALTO_CHAIN_ID,
-				}
```

Comment:
I think this is not needed anymore since my understanding is that it was only used for the UI.